facet/README.md

```diff
@@ -0,0 +1,205 @@
+--- 
+layout: pattern
+title: Facet
+folder: facet
+permalink: /patterns/facet/
+categories:
+- structural
+language: en
+tags:
+- Decoupling
+---
+
+## Also known as
+
+Attenuation
+
+## Intent
+
+Provide an interface to a powerful object in a restricted way, either by restricting parameters 
+or only allowing calls to a subset of object's functions such that the powerful object's 
+capabilities are not abused by any other classes.
+
+## Explanation
+
+Real-world example
+
+> Imagine a knight fighting a dragon. The knight can only attempt to attack the dragon by trying
+> different attacks. Here, the facet stops the knight from directly changing the health of the 
+> dragon, as well as only allowing certain attacks to affect the dragon.
+
+In plain words
+
+> Using the facet pattern, a facet class represents strictly limited access to another class.
```

Comment:
I think this could be more clear, im not quite sure what this is trying to communicate without prior information